packages/account/codegen.json

```diff
@@ -30,7 +30,8 @@
           "UtxoId": "string"
         },
         "useTypeImports": true,
-        "nonOptionalTypename": true,
+        "dedupeOperationSuffix": true,
+        "skipTypename": true,
         "typesPrefix": "Gql"
```

Comment:
Should we remove this as well? seems like it just ends up [becoming unnecessary](https://github.com/FuelLabs/fuels-ts/pull/2241/files#diff-69a6670d0426c74935ed2830ce552e7df2379c2d112a582306e57569fcd17029R98-R103)

Reply:
@maschad I would prefer not to since this prefix indicates that this type is related to GraphQL.

Also, we do not use [this approach](https://github.com/FuelLabs/fuels-ts/pull/2241/files#diff-69a6670d0426c74935ed2830ce552e7df2379c2d112a582306e57569fcd17029R98-R103) for every GraphQL type